### PR TITLE
fix: Add optional chaining for 'isArtificalRecoveredEventItem' (no-changelog)

### PR DESCRIPTION
**Pull request by alexgrozav**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

File: packages/editor-ui/src/mixins/pushConnection.ts [hunk 1]
```diff
@@ -295,7 +295,7 @@ export const pushConnection = mixins(
 						for (const key of Object.keys(activeRunData)) {
 							if (
 								pushData.data.data.resultData.runData[key]?.[0]?.data?.main?.[0]?.[0]?.json
-									.isArtificalRecoveredEventItem === true &&
+									?.isArtificalRecoveredEventItem === true &&
```
Comment (netroy) [inline packages/editor-ui/src/mixins/pushConnection.ts:298]: while we are at it, can you please rename all typos `Artifical` to `Artificial` across the repo 🙏🏽

**Review by netroy — CHANGES_REQUESTED** [review]

File: packages/editor-ui/src/mixins/pushConnection.ts (hunk 1 above)
Comment (alexgrozav) [reply packages/editor-ui/src/mixins/pushConnection.ts]: Good catch! Done.

**Review by netroy — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@0.222.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.222.1)

**Comment by janober** [comment]
Got released with [`n8n@0.222.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.222.1)

**Pull request merged**